iOSDemo/DemoViewController.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+import UIKit
+import SweetUIKit
+
+enum DemoItem: Int {
+    case edit, collection
+    
+    static var titles: [String] {
+        let allItems: [DemoItem] = [.edit, .collection]
+        return allItems.map{ $0.title }
+    }
+    
+    var title: String {
+        switch self {
+        case .edit: return "Edit Controller"
+        case .collection: return "Collection Controller"
+        }
+    }
+    
+    var viewController: UIViewController {
+        switch self {
+        case .edit: return EditViewController()
+        case .collection: return CollectionController()
+        }
+    }
+}
+
+class DemoViewController: SweetTableController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableView.register(TableViewCell.self)
```

Comment:
We are using `self` for instance variables. 

So `self.tableView` all the things.